### ISSUE TITLE
[FEAT] New service: cURL / Webhook endpoint

### ISSUE THREAD

**Issue by kayvanaarssen** [tigattack/VeeamNotify#69]
### Is your feature request related to a problem?

cURL / Webhook

### Describe your ideal solution

Send the log data etc. To webhooks or cURL to present the data in external tools. We have a monitoring section in our custom portal. We feed it from our linux backups with cURL uploads to a signed url in Laravel. And extract the data. Would be nice to have this option.

**Comment by tigattack** [comment]
I like this idea. I assume, instead of log data, you mean the session information? It would be trivial to just send the gathered data in a flat JSON structure to a specified endpoint, I can certainly look into this.

**Comment by kayvanaarssen** [comment]
Yes, i’ve played around with the RestAPI some time ago just for testing. But you will get usefull data back from it to send to the channels.

**Comment by tigattack** [comment]
Added in [v2.0.0](https://github.com/tigattack/VeeamNotify/releases/tag/v2.0.0).
